aggregateur/main.py

```diff
@@ -109,7 +110,7 @@ def __init__(self, git_url, email, schema_type):
         self.owner = self.find_owner(parsed_git)
         self.name = parsed_git.name
         self.email = email
-        self.git_repo = None
+        self.git_repo = GitRepo(self.clone_dir)
```

Comment:
Don't initialize this variable to `None`. In case of a timeout, it's useful to point the `git_repo` to the target clone directory: repositories are cached between CI executions, therefore the old content will already be present. If clone / pull is not possible, use the old content to validate tags and schemas